stock_inventory_discrepancy/models/stock_location.py

```diff
@@ -15,3 +15,22 @@ class StockLocation(models.Model):
         "an Inventory Adjustment. Thresholds defined in Locations have "
         "preference over Warehouse's ones.",
     )
+    update_child_location = fields.Boolean(
+        string="Propagate to child locations",
+        help="Propagate Maximum Discrepancy Rate Threshold to child locations",
+    )
```

Comment:
I see the naming very generic. IMO something like `propagate_discrepancy_threshold` and a string "Propagate discrepancy threshold" would be more appropriate.

Reply:
Done!